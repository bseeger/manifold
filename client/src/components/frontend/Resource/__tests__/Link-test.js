import React from 'react';
import renderer from 'react-test-renderer';
import Link from '../Link';
import build from 'test/fixtures/build';

describe("Frontend.Resource.Link component", () => {

  const resource = build.entity.resource("1");

  it('renders correctly', () => {
    const component = renderer.create(
      <Link
        attributes={resource.attributes}
      />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
