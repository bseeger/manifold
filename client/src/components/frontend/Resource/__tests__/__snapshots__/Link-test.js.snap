exports[`Frontend.Resource.Link component renders correctly 1`] = `
<a
  className="button-primary"
  onClick={[Function]}
  style={Object {}}
  target="_blank">
  Download 
  <i
    className="manicon manicon-arrow-down" />
</a>
`;
